src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -107,6 +107,13 @@ else()
 endif()
 
 if (CLR_CMAKE_TARGET_APPLE)
+    if(CLR_CMAKE_TARGET_IOS)
+        add_definitions(-DTARGET_IOS)
+    elseif(CLR_CMAKE_TARGET_TVOS)
+        add_definitions(-DTARGET_TVOS)
+    elseif(CLR_CMAKE_TARGET_MACCATALYST)
+        add_definitions(-DTARGET_MACCATALYST)
+    endif()
```

Comment:
Does the Apple hybrid globalization depend on the differences between iOS, tvOS and macCatalyst anywhere?

If not, would it be better to define `APPLE_HYBRID_GLOBALIZATION` symbol for the 3 OSes and use it throughout the implementation instead of `defined(TARGET_MACCATALYST) || defined(TARGET_IOS) || defined(TARGET_TVOS)` conditions?

Reply:
Hybrid globalization is same for iOS, tvOS and macCatalyst. Created separate ticket for it https://github.com/dotnet/runtime/issues/96328 as this PR may not be merged.